org.aardappel.TreeSheets.json

```diff
@@ -0,0 +1,100 @@
+{
+    "app-id": "org.aardappel.TreeSheets",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "/app/treesheets",
+    "branch": "master",
+    "tags": ["master", "notes", "wxWidgets"],
+    "build-options": {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g",
+        "env": {
+            "V": "1"
+        },
+        "build-args": [
+            "--share=network"
+        ]
+    },
+    "cleanup": [
+        "/include",
+        "/tests",
+        "/man",
+        "/share/man",
+        "treesheets.desktop",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "wxWidgets",
+            "config-opts": [
+                "--enable-unicode"
+            ],
+            "cxxflags": [
+                "-std=c++17"
+            ],
+            "make-args": [
+                "-j4"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/wxWidgets/wxWidgets/releases/download/v3.1.0/wxWidgets-3.1.0.tar.bz2",
+                    "sha256": "e082460fb6bf14b7dd6e8ac142598d1d3d0b08a7b5ba402fdbf8711da7e66da8"
+                }
+            ]
+        },
+        {
+            "name": "treesheets",
+            "buildsystem": "cmake",
+            "make-args": [
+                "-j8"
+            ],
+            "post-install": [
+                "install -Dm644 TS/images/icon16.png /app/share/icons/hicolor/16x16/apps/org.aardappel.TreeSheets.png",
+                "install -Dm644 TS/images/icon32.png /app/share/icons/hicolor/32x32/apps/org.aardappel.TreeSheets.png",
+                "install -m755 TS/treesheets /app/"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/aardappel/treesheets"
```

Comment:
Flathub only accepts versioned and stable releases. If this project has neither then this package can wait or you can worth with upstream to get a release maybe.